templates/settings/payment.php

```diff
@@ -10,44 +10,113 @@
+        <h2> <?php esc_html_e( 'Payment Methods', 'dokan-lite' ); ?></h2>
+        <div id="vendor-dashboard-payment-settings-toggle-dropdown">
+            <button id="toggle-vendor-payment-method-drop-down" class="dokan-btn dokan-btn-success"> <?php esc_html_e( 'Add Payment Method', 'dokan-lite' ); ?></button>
+            <div id="vendor-payment-method-drop-down-wrapper">
+                <div id="vendor-payment-method-drop-down">
+                    <?php
+                    $has_methods_in_dropdown = false;
+                    $icon_mismatch_map       = [
+                        'dokan-stripe-connect'     => 'stripe',
+                        'dokan-paypal-marketplace' => 'paypal'
+                    ];
+                    $mis_match_methods       = [ 'stripe', 'moip' ];
+                    $unused_methods          = get_unused_payment_methods( $methods, $profile_info['payment'], $mis_match_methods );
+                    ?>
+                    <ul>
+                    <?php foreach ( $unused_methods as $method_key ) :
+                        $method = dokan_withdraw_get_method( $method_key );
 
-    <?php wp_nonce_field( 'dokan_payment_settings_nonce' ); ?>
+                        if ( ! empty( $method ) ) {
+                            $has_methods_in_dropdown = true;
+                        } else {
+                            continue;
+                        }
+                        ?>
+                        <li>
+                            <a href="<?php echo esc_url( home_url( "dashboard/settings/payment/manage-" . $method_key ) ); ?>">
+                                <div>
+                                    <img src="<?php echo esc_url( dokan_withdraw_get_method_icon( isset( $icon_mismatch_map[ $method_key ] ) ? $icon_mismatch_map[ $method_key ] : $method_key ) ); ?>" alt="<?php echo esc_attr( $method_key ); ?>" />
+                                    <span>
+                                        <?php
+                                        //translators: %s: payment method title
+                                        echo sprintf( __( 'Direct to %s', 'dokan-lite' ), esc_html__( $method['title'], 'dokan-lite' ) );
+                                        ?>
+                                    </span>
+                                </div>
+                            </a>
+                        </li>
+                    <?php endforeach; ?>
+                    </ul>
+
+                    <?php if ( ! $has_methods_in_dropdown ) : ?>
+                        <div>
+                            <?php esc_html_e( 'All payment methods are used.', 'dokan-lite' ); ?>
+                        </div>
+                    <?php endif; ?>
+                </div>
+            </div>
+        </div>
+    </div>
+    <ul>
+
+    <?php
+    wp_nonce_field( 'dokan_withdraw_make_default' );
+    $default_method = dokan_withdraw_get_default_method( $current_user );
+    $mis_match_map  = [
```

Comment:
Declare all these required variables in the function where the template is being included. And if this is being included using `dokan_pro_get_template`, then pass all these variables in the `$args`